src/assets/style.scss

```diff
@@ -181,9 +181,18 @@ pre {
 
 table {
   border-collapse: collapse;
+  margin-bottom: $padding-containers;
+
   td, th {
     padding: $padding-containers-small;
     border: 1px solid $border-color;
+
+    p:first-child {
+      margin-top: 0;
+    }
+    p:last-child {
+      margin-bottom: 0;
+    }
```

Comment:
Table cells have padding. There's not really a need for leading and trailing margins for flow content within them as it makes the cells look too tall.